ipwb/replay.py

```diff
@@ -548,8 +550,11 @@ def handler(signum, frame):
         print(traceback.format_exc())
         print(sys.exc_info()[0])
     except HashNotFoundError:
-        print("Hashes not found")
-        return '', 404
+        if payload == '':
```

Comment:
@machawk1, payload could legitimately be empty, so assuming that an empty string means it was payload that was missing may lead to some wrong conclusions while actually the header might have had the issue in being fetched from the IPFS.

Reply:
@ibnesayeed See #378